deploy/crd.yaml

```diff
@@ -134,7 +134,7 @@ spec:
                 replicas:
                   type: integer
                   minimum: 1
-                offHeapMemoryFraction:
+                offProcessMemoryFraction:
```

Comment:
For backwards compatibility, we'll need to also keep the existing "offHeapMemoryFraction" field in the CRD definition and throughout the codebase. Otherwise, this change will break existing users who are relying on that. We can mark it as deprecated in the docs, and if both are specified we can ignore offHeap. See what we've done for 

 We can mark it deprecated in the docs though.

Naming-wise, maybe "nonFlinkMemoryFraction"? Or we could invert it, and have it as something like "systemMemoryFraction."

Reply:
Can you take a look at the new changes? I think it should be ok now :)